examples/faraday_redirect_options.rb

```diff
@@ -0,0 +1,32 @@
+# A basic MetaInspector example for scraping a page
+#
+# Usage example:
+#
+#   ruby faraday_redirect_options.rb http://facebook.com
```

Comment:
Can you explain a bit more about what this example does? It's not a basic example, it's rather advanced.

I think the reader might need some context to understand what this example does (raising an exception when following a redirection to a private IP address), but using Facebook will never demonstrate the point.

We could either document how to craft a redirection to a private IP address, or make a simpler example that uses the callback for something simpler like for example logging the URLs.